build/Build.cs

```diff
@@ -25,7 +25,7 @@ class Build : NukeBuild
     // ARGUMENTS
     //////////////////////////////////////////////////////////////////////
     [Parameter] readonly string Configuration = "Release";
-    [Parameter] readonly string SigningCertificatePath = "./certificates/OctopusDevelopment.pfx";
+    [Parameter] readonly string SigningCertificatePath = RootDirectory / "certificates" / "OctopusDevelopment.pfx";
```

Comment:
Ooh! Shiny Path.Combine operator from NUKE. I didn't know about this :)

Reply:
Yeah, tis very nice.